NEWTASK1.js

```diff
@@ -0,0 +1,87 @@
+//TEST CASES
+const movies=[
+				{
+					name:'Lion king',
+					license: [2008,2010]
+				},
+				{
+					name:'Harry potter',
+					license: [2003,2019]
+				},
+				{
+					name:'Ice Age',
+					license: [2004,2015]
+				},
+				{
+					name:'Up',
+					license: [2007,2011]
+				},
+				{
+					name:'Cars',
+					license: [2006,2009]
+				}
+			];
+
+function yearWithMaxNumOfMovies(movieList){

```

Comment:
Break down this giant function into smaller logical pieces, kind of like: https://github.com/Real-Software-Dev-Squad/Max-Movies-In-Years/pull/2#issuecomment-599252338